src/motor-operations.h

```diff
@@ -64,22 +69,30 @@ class MotorOperations {  // Rename SegmentQueue ?
 
   // Wait, until all elements in the ring-buffer are consumed.
   virtual void WaitQueueEmpty() = 0;
+
+  // Get a reference of a FixedArray containing the absolute motors
+  // positions in steps.
+  virtual const MotorCoordSystem GetCurrentMotorPos() = 0;
 };
 
 class MotionQueueMotorOperations : public MotorOperations {
 public:
   // Initialize motor operations, sending planned results into the motion backend.
-  MotionQueueMotorOperations(MotionQueue *backend) : backend_(backend) {}
+  MotionQueueMotorOperations(MotionQueue *backend);
+  ~MotionQueueMotorOperations();
```

Comment:
Even though not needed, it is always a good idea to prepend this with 'virtual' for the user to read (once we decide to use c++11 flags, we can have 'override' instead at the end, but we don't have that at this point).